CODEOWNERS

```
@@ -5,3 +5,6 @@
 # Order is important: later rules override preceding rules
 
 * @mesosphere/sig-ksphere-catalog
+
+# Prometheus
+addons/prometheus/* @gracedo
```

These are not additive, the more specific rule replaces the global rule. So we'd prefer this:
```suggestion
addons/prometheus/* @mesosphere/sig-ksphere-catalog @gracedo
```